ReginaGyimesi/CodeToGive2022_Frontend
# Add Custom Slider

## What does this PR do?
- Create new component Slider, for work motivation test, as now its a 1-5 question type
## How should it be tested?
- Go to client/work-motivation
- Set value by dragging. Click on next arrow should trigger form submit console.log(), can be seen in console
